drivers/src/persistent.rs

```diff
@@ -74,6 +76,17 @@ pub struct PersistentData {
 
     #[cfg(not(feature = "runtime"))]
     pcr_reset: [u8; memory_layout::PCR_RESET_COUNTER_SIZE as usize],
+
+    #[cfg(feature = "runtime")]
+    pub rt_pub_key: Ecc384PubKey,
+    #[cfg(feature = "runtime")]
+    pub attestation_disabled: U8Bool,
+    #[cfg(feature = "runtime")]
+    reserved8: [u8; memory_layout::DISABLE_ATTESTATION_SIZE as usize
+        - size_of::<Ecc384PubKey>()
+        - size_of::<U8Bool>()],
+    #[cfg(not(feature = "runtime"))]
+    rt_pub_key: [u8; memory_layout::DISABLE_ATTESTATION_SIZE as usize],
```

Comment:
If we do the above, we can probably just modify the FHT in-place (or store in the drivers struct). Since we'll be updating it on every boot anyway.

Reply:
Done